packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -37,40 +39,47 @@ export class AddJob {
   @InstrumentUsecase()
   @LogDecorator()
   public async execute(command: AddJobCommand): Promise<void> {
-    Logger.verbose('Getting Job');
+    Logger.verbose('Getting Job', LOG_CONTEXT);
     const job =
       command.job ?? (await this.jobRepository.findById(command.jobId));
-    Logger.debug(job, 'job contents');
+    Logger.debug(`Job contents for job ${job._id}`, job, LOG_CONTEXT);
 
     if (!job) {
-      Logger.warn('job was null in both the input and search');
+      Logger.warn(
+        `Job ${job._id} was null in both the input and search`,
+        LOG_CONTEXT
+      );
 
       return;
     }
 
-    Logger.log('Starting New Job of type: ' + job.type);
+    Logger.log(`Starting New Job ${job._id} of type: ${job.type}`, LOG_CONTEXT);
 
     const digestAmount =
       job.type === StepTypeEnum.DIGEST
         ? await this.addDigestJob.execute(AddDigestJobCommand.create({ job }))
         : undefined;
-    Logger.debug('digestAmount is: ' + digestAmount);
+    Logger.debug(`DigestAmount is: ${digestAmount}`, LOG_CONTEXT);
 
     const delayAmount =
       job.type === StepTypeEnum.DELAY
         ? await this.addDelayJob.execute(command)
         : undefined;
-    Logger.debug('delayAmount is: ' + delayAmount);
+    Logger.debug(`DelayAmount is: ${delayAmount}`, LOG_CONTEXT);
 
     if (job.type === StepTypeEnum.DIGEST && digestAmount === undefined) {
-      Logger.warn('Digest Amount does not exist on a digest job');
+      Logger.warn(
+        `Digest Amount does not exist on a digest job ${job._id}`,
+        LOG_CONTEXT
+      );
 
       return;
     }
 
     if (digestAmount === undefined && delayAmount === undefined) {
       Logger.verbose(
-        'updating status as digestAmount and delayAmount is undefined'
+        `Updating status as digestAmount and delayAmount is undefined for job ${job._id}`,
+        LOG_CONTEXT
```

Comment:
I feel this relays an incorrect state - if I understand correctly, here it realizes it is not a delay or a digest, so its a normal step and he updates its status to Queued like it should. 
Maybe we could add to the message that the 'Updating status' is to queued

Reply:
Yeah, we can simplifiy the message.